pkg/server/server.go

```diff
@@ -389,8 +393,19 @@ func (s *Server) HTTPHandler() http.Handler {
 	handle("/api/console/knative-channels", authHandler(s.handleKnativeChannelCRDs))
 	handle("/api/console/version", authHandler(s.versionHandler))
 
+	// Get running kubernetes version and inject into helm handlers.
+	config := &rest.Config{
+		APIPath:   s.K8sProxyConfig.Endpoint.String(),
+		Transport: s.K8sClient.Transport,
+	}
+	kubeVersion, err := actions.GetKubeVersion(config)
+	if err != nil {
+		klog.Warningf("Failed to get cluster k8s version %s", err.Error())
```

Comment:
We'll want to make sure this is logged by default. @jhadvig Do you know offhand?

Reply:
We should fall back to the env var at least on failure.